semcore/data-table/src/Body.tsx

```diff
@@ -58,12 +59,59 @@ class Body extends Component<AsProps, State> {
   firstRowRef = React.createRef<HTMLDivElement>();
   firstRowResizeObserver: ResizeObserver | null = null;
 
+  lockedCell: [HTMLElement | null, boolean] = [null, false];
+
   getRowHeight = () => {
     const { virtualScroll } = this.asProps;
     const rowHeightFromProps = typeof virtualScroll === 'object' && virtualScroll?.rowHeight;
     return rowHeightFromProps || this.state.rowHeight;
   };
 
+  handleKeyDown = (e: React.KeyboardEvent) => {
+    if (e.currentTarget === this.lockedCell[0]) {
+      const focusableChildren = Array.from(this.lockedCell[0].children).flatMap((node) =>
+        getFocusableIn(node as HTMLElement),
+      );
+
+      if (this.lockedCell[1]) {
+        if (e.key === 'Escape') {
+          this.lockedCell[0]?.focus();
+          this.lockedCell[1] = false;
+        }
+        if (e.key.startsWith('Arrow')) {
+          e.stopPropagation();
```

Comment:
I guess you should do prevent default here as well. Otherwise the page scrolls alongside the navigation happens.

Reply:
fixed